### PR TITLE
Move to codes as primary keys for the models.

### PR DESCRIPTION
**Pull request by plural**
Update the importer to deal with all these changes.
Remove legality models since we don't have a fleshed out design for formats and banlists yet.

### CODE REVIEW

File: app/models/card_subtype.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CardSubtype < ApplicationRecord
+  self.table_name = "cards_subtypes"
+
+  belongs_to :cards,
+    :class_name => "Card",
```
Comment (NoahTheDuke) [inline app/models/card_subtype.rb:7]: Still needed?

**Review by NoahTheDuke — APPROVED** [review]

File: app/models/card_subtype.rb (hunk 1 above)
Comment (plural) [reply app/models/card_subtype.rb:7]: yep. If i remove the class names i get these errors when using the associations:

 Card Load (12.7ms)  SELECT "cards".* FROM "cards"
/usr/local/bundle/gems/activerecord-7.0.1/lib/active_record/reflection.rb:435:in `rescue in compute_class': Rails couldn't find a valid model for Subtypes association. Please provide the :)

          raise NameError, msg
          ^^^^^
/usr/local/bundle/gems/activerecord-7.0.1/lib/active_record/inheritance.rb:259:in `compute_type': uninitialized constant Card::Subtypes (NameError)

            raise NameError.new("uninitialized constant #{candidates.first}", candidates.first)
            ^^^^^
irb(main):002:0> Subtype.all[5].cards
  Subtype Load (4.7ms)  SELECT "subtypes".* FROM "subtypes"
/usr/local/bundle/gems/activerecord-7.0.1/lib/active_record/reflection.rb:435:in `rescue in compute_class': Rails couldn't find a valid model for Cards association. Please provide the :cla)
                                                                            
          raise NameError, msg                                              
          ^^^^^                                                             
/usr/local/bundle/gems/activerecord-7.0.1/lib/active_record/inheritance.rb:259:in `compute_type': uninitialized constant Subtype::Cards (NameError)
                                                                            
            raise NameError.new("uninitialized constant #{candidates.first}", candidates.first)
            ^^^^^                                                           
irb(main):003:0>

Comment (NoahTheDuke) [reply app/models/card_subtype.rb:7]: Hmmmmmmmmmmmmmm! Something is broken here but I don't know what. Don't let this hold you up so merge away. I'll take a look when I get my new computer set up.

Comment (plural) [reply app/models/card_subtype.rb]: oh!  Number matters!  belongs_to :singular_form takes care of this. Thanks for the review!

**Pull request merged**